README.md

```diff
@@ -312,3 +360,75 @@ reported properly.
 
 *NOTE*: the current implementation does not handle all edge cases at this time.
 If you send invalid data, it is possible for it to silently fail.
+
+## How to fork this add-on to create your own SudoSandwich instance on Heroku
+
+If you want to run your own add-on based on this codebase, you should follow these instructions. These instructions are for deploying your add-on to Heroku, which is not required. You may have to modify the deploy instructions for other environments.
+
+To start, you may wish to fork the codebase to your own private GitHub repo, so that you can make changes for your specific add-on's use case. If you intend to do local development, see [Set up](#set-up).
+
+Create a Heroku app. Typically, we name the Heroku app after the slug (or command line identifier) for the add-on. If I was creating an add-on with the slug of `sudo-sandwich`, I'd probably name the Heroku app `sudo-sandwich` as well. This is not required; and you can name it whatever you'd like. Please note that `sudo-sandwich` is already taken, and slugs are both immutable and must be globally unique!
+
+Install the Heroku CLI, if you haven't already, [from the instructions for your platform](https://devcenter.heroku.com/articles/heroku-cli#download-and-install).
+
+```
+heroku apps:create your-app-slug
+```
+
+You'll need a Heroku Postgresql database to store data in:
+
+```
+heroku addons:create heroku-postgresql --app your-app-slug
+```
+
+You will also want to scale the `worker` dyno up so that async provisioning can be handled in the background:
+
+```
+heroku ps:scale worker=1 --app your-app-slug
+```
+
+Deploying the code to Heroku either [using git](https://devcenter.heroku.com/articles/git) or [the Heroku Dashboard with the GitHub integration](https://devcenter.heroku.com/articles/github-integration).
+
+Generate a manifest with the [addons-admin](https://github.com/heroku/heroku-cli-addons-admin) CLI plugin. (See the linked repo for plugin install instructions.) After install, run `heroku addons:admin:manifest:generate` and follow the prompts.
```

Comment:
seems like in this section you are doing inline markdown links but above you were listing links below. Not a big deal but it might be nice to be internally consistent on which style. I don't have a strong preference either way

Reply:
oh was the setup guide existing? This diff makes it look like you added, for example, `[Ruby on Rails]: http://rubyonrails.org`

Reply:
Ah right, that content came out of `CONTRIBUTING.md` which I have not changed/removed yet.

I didn't get an answer for a `CONTRIBUTING.md` we use for all @heroku open source repos, so we'll have to put something basic in there for now.

Sorry, I'd totally forgotten about that! Changes were made too long ago.

Reply:
no biggie! happy to keep as-is for now if you just want to :shipit: 

this doc is great

Reply:
Cool. I'd like to keep improving it, but it will take more iterations :)